README.md

```diff
@@ -490,6 +491,21 @@ DESCRIPTION
 
 _See code: [src/commands/logout.js](https://github.com/apify/apify-cli/blob/v0.16.2/src/commands/logout.js)_
 
+## `apify pull [ACTORID]`
+
+Pulls the latest version of an actor from the Apify platform to the current directory.
+
+```
+USAGE
+  $ apify pull [ACTORID]
+
+ARGUMENTS
+  ACTORID  ID or name (username/actor_name, ~actor_name for logged user) of an existing actor on the Apify platform
```

Comment:
Should it be same as ACTORID in apify push command?
It will make sense to do it same.